index.html

```diff
@@ -66,20 +67,26 @@ <h2 class="secondary-text">I'm a Software Developer</h2>
                             <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
                         </ul>
                     </div>
-                    <img class="background-image" src="./assets/images/Illustration.svg" alt="header-banner">
+                    
                 </div>
-
-    
+                <img class="background-image" src="./assets/images/Illustration.svg" alt="header-banner">
+                <img src="./assets/images/desktop header2.svg" alt="desktop-banner" class="dekstop-banner">
```

Comment:
- [ ] As requested by the previous reviewer, kindly use those as the background images for the whole `headline` section.
- Instead of what you are using now, I found you used the grey color as the `background-image` for the whole section, which is unnecessary, you should have used the grey color as a `background-color` for the entire body as the whole project uses the same color, and for the contact form you can specify its own color, by doing that, there will be a room for you to use the illustrations as a `background-image` for the whole `headline` section.